screen-redraw.c

```diff
@@ -31,57 +31,9 @@ static void	screen_redraw_draw_pane(struct screen_redraw_ctx *,
 static void	screen_redraw_set_context(struct client *,
 		    struct screen_redraw_ctx *);
 
-#define CELL_INSIDE 0
```

Comment:
ℹ️ The removed code here was put into `tmux.h` so it could also be used by `screen-write.c`